Update Top Cover Installation

jaysuk opened the pull request:
Rewrite the top cover installation page, detailing all steps required, with exploded CAD images.
Fastener information updated to make it clearer about the type to be installed.
STL links updated to the correct files.
Page renamed from top cover latches to top cover

jaysuk committed:
Rewrite top cover installation page

update STL links
new images
updated fastener descriptions

HumbleDeer commented on docs/vz235_mellow/top_cover/top_cover.md:
```diff
@@ -0,0 +1,77 @@
+| [Top cover middle part]        | 4               |
+| [Top cover top part]           | 4               |
+
+### Step 1
```
The headings look rather small with the h3 heading size. Would h2 solve this, or could a styling change be necessary to make the steps clearer to the user? 

Been considering making headings "looser" and have more margin above them if they are not the first one in the page.

jaysuk replied:
I used the same size that was already there.
Not sure what you mean about making them "looser". The spacing of them is as per the markdown standard

HumbleDeer replied:
This comment was mostly conversation oriented. My bad for not communicating that clearly.

HumbleDeer commented on docs/vz235_mellow/top_cover/top_cover.md:
```diff
@@ -0,0 +1,77 @@
+has_toc: false
+nav_order: 1
+has_children: false
+permalink: /vz235_mellow/top_cover/latches
```
The permalink here doesn't match the name of the file, which could cause problems; It's recommended for them to (at least somewhat) match between the FS and the web pages.

You can replace `permalink:` with `slug:` to define the name of one page, with the rest of the navigation structure determined by the folder names.

Another option is leaving both out entirely, which would autogenerate the URL entirely from the filesystem.

Both `has_toc` as well as `has_children` can be left out if they are false; we do not have them true by default and they are false by default according to the JustTheDocs defaults. These pages can't have children as of now, regardless. Two levels is the max.

jaysuk replied:
updated

HumbleDeer commented on docs/vz235_mellow/top_cover/top_cover.md:
```diff
@@ -0,0 +1,77 @@
+
+### Step 4
+
+![Handle Scews](../../assets/images/manual/vz235_mellow/top_cover/handle_screws.png)
```
Could you change this picture's Alt text so it is not the same as step 6's picture? The alt text serves accessibility purposes, so a descriptive name is useful.

I personally use assistive technologies, so I've got an eye for things like these. :)

jaysuk replied:
Updated

HumbleDeer commented on docs/vz235_mellow/top_cover/top_cover.md:
```diff
@@ -0,0 +1,77 @@
+---
+layout: default
+title: 8.1 Top cover
```
A description could be useful here. They are shown in embeds and search results. Not a must, but I'd recommend it.

jaysuk replied:
None of the other pages seem to have a description.
How would you like me to add one?

HumbleDeer replied:
I'm trying to add descriptions to more pages, besides the few I've already done. Something descriptive of the general topic is fine; I suppose mentioning the specific printer could be important as the previews don't actually include the grandparent name

HumbleDeer replied:
Looks good! Cheers.

HumbleDeer commented on docs/vz235_mellow/top_cover/top_cover.md:
```diff
@@ -0,0 +1,77 @@
+| M4 x 10 SHCS                | 8        |                                                      |
+| M4 hammer t-nut             | 8        |                                                      |
+| 2 x 8mm Magnet              | 8        |                                                      |
+| 395\*405\*3mm acrylic panel | 1        | Refer to the .step file for placement of the cutouts |
```
One can also refer to the web Fusion360 model. The CNC and Printed versions are both linked at the top of the README.MD here: https://github.com/VzBoT3D/VzBoT-Vz235

jaysuk replied:
I can refer to that. I had just left the wording the same as what was already there.

jaysuk replied:
I have decided to remove that note as this is a kit after all and its supplied fully manufactured.

HumbleDeer reviewed (changes requested):
Great work! Thanks!

I'm approving of these changes, because they vastly improve the page. Besids the small comments, I do not have any other objections.
The code respects the Markdown conventions and is tidy. me like.

One more thought: Does the Vz235 Printed version not match the CNC version almost entirely? If they do, we could change this PR to merge into a different branch for now which can then be merged into gh-pages alltogether as "Vz235 Top cover" stuff. 
-- If that makes things more inconvenient, we can merge it here too. I prefer not clouding the history tree with too many one word commits, as it makes it really *really* ***really*** hard to debug something and/or find where a change was made.

HumbleDeer commented on docs/vz235_mellow/top_cover/top_cover.md:
```diff
@@ -0,0 +1,77 @@
+
+## Overview
+
+![Overview](../../assets/images/manual/vz235_printed/top_cover/top_cover_overview.png)
```
This image doesn't really represent the top cover at a first glance. Ditto for the vz235 printed picture, as used. 🤔 Does the acrylic/PC panel show up properly for screenshots?

In general, all the URL's for assets can be referred to from the root directory of the docs folder we build from (./docs from repo root, i.e. the second docs folder).

Relative URL's are not really necessary here; it may also increase the risk of frustration with the URL's, and can make the source code untidy when they become six or seven iterations deep. 😳

jaysuk replied:
I hadn't bothered changing the image that was already on the page. I may just remove the overview image entirely.
For the URL's I was just replicating what was already in use on the page.

jaysuk replied:
I have removed the overview image and left the URL's as they as as thats the way they are done on every other page I looked at

HumbleDeer commented:
Somewhat related to https://github.com/VzBoT3D/docs/pull/43 (Vz330)

jaysuk committed:
Update a few things based on feedback

Fixed permalink, changed heading size, removed note to look at step file for panel, centred quantities in table.

jaysuk committed:
Add a description to the top cover installation page

HumbleDeer reviewed (approved):
All comments have been resolved, so I have no objections to merging. 

If you'd like to add the time changed to the front matter, feel free to do that here. :)

jaysuk committed:
Add update date to the front matter
